x-pack/plugins/cases/server/services/user_profiles/index.ts

```diff
@@ -75,7 +73,7 @@ export class UserProfileService {
 
       UserProfileService.validateSizeParam(size);
 
-      if (!UserProfileService.isSecurityEnabled(securityPluginFields)) {
+      if (!UserProfileService.isSecurityEnabled(securityPluginFields) || owners.length <= 0) {
```

Comment:
If `owners` is an empty array, then we could potentially return users who don't have access for cases. @cnasikas can you think of a scenario where the `owners` field might not be passed to the APIs? I think I covered the flyout and model scenarios with my `useAvailableCasesOwners()` call.

Reply:
I think it is fine as long as we pass the owners ourselves to the API when the consumer of the cases passes an empty array as `owner` on the `CasesContext`.